extensions/notebook/src/test/book/book.test.ts

```diff
@@ -755,10 +755,12 @@ describe('BooksTreeViewTests', function () {
 					should(showNotebookSpy.calledWith(vscode.Uri.file(notebookPath))).be.true(`Should have opened the notebook from ${notebookPath} in the editor.`);
 				});
 
-				it('openNotebookAsUntitled should open a notebook as untitled file in the editor @UNSTABLE@', async () => {
+				it('openNotebookAsUntitled should open a notebook as untitled file in the editor', async () => {
```

Comment:
Should we really be removing this from unstable already? It may be better to remove from unstable only after it succeeds for some amount of time (e.g. a week)

Reply:
It's easy to forget about stuff like this, I'd rather we just promote it back and if it fails again we can demote it and keep looking. One build failure isn't a huge deal IMO.